custom-elements.json

```diff
@@ -9,11 +9,11 @@
+          "default": "class _IncludeFragmentElement extends HTMLElement {\n  static"
```

Comment:
Not sure if this is unusual how it's cut off at `static"`?

Reply:
ugh idk why this file changed at all

Reply:
I'm not really sure if it's important? like why would we store code in a json object 🤔